pkg/operator/k8sutil/pod.go

```diff
@@ -34,8 +35,19 @@ const (
 	PodIPEnvVar = "ROOKD_PRIVATE_IPV4"
 )
 
+func RepoPrefix() string {
+	defaultRepoPrefix := "quay.io/rook"
+
+	var repoPrefix string
+	if repoPrefix = os.Getenv("ROOK_OPERATOR_REPO_PREFIX"); repoPrefix == "" {
```

Comment:
also would be good to have the env var in a const so the unit tests can reference it too